### ISSUE TITLE
Build: Consider merging debian/control into a single file

### ISSUE THREAD

**Issue by hoffie** [jamulussoftware/jamulus#2430]
**Has this feature been discussed and generally agreed?**

No.

**Describe the solution you'd like**

We should check if we can merge the two control files to avoid confusion. IIRC the one was supposed to be "official", but I think it's maintained on the Debian-repo side anyway.

**Describe alternatives that have been considered**

**Comment by pljones** [comment]
Should this be done as part of the larger task of hosting a package archive for the Jamulus project?

**Comment by hoffie** [comment]
> Should this be done as part of the larger task of hosting a package archive for the Jamulus project?

Hmm, besides being also about "Debian packaging" I'm not sure if they are that closely related? I think both tasks could be done indepdendently.

**Comment by pljones** [comment]
Yep, there's a few related tasks (like getting the source .deb done, too).  They need coordinating but can be done independently.

**Comment by ann0see** [comment]
I think we should have a look at the control file anyway. It’s somewhat hacked together as I copied the dependencies from another system to allow installation on Debian 10 and don’t use the automated sharedlibs (?) command

**Comment by hoffie** [comment]
> don’t use the automated sharedlibs (?) command

... which might be hacky, but is probably the reason why the build works on Ubuntu 18.04 despite the known problems with the Github Runner Ubuntu 18.04 images (non-standard GCC versions).

**Comment by ann0see** [comment]
Maybe even remove the build-debian-package script and move the "auto" one into the Linux folder
